bitnami/kafka/Chart.yaml

```
@@ -40,4 +40,4 @@ maintainers:
 name: kafka
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/kafka
-version: 31.4.1
+version: 31.4.2
```

Sorry Yashmina I didn't see this before, I tried to do it by myself but I don't have permissions in your repository.

Your are adding a new feature so, according to semver.org, you should bump a minor version

```suggestion
version: 31.5.0
```

---

No worries, I've changed. Thanks @fmulero